apps/client/src/app/pages/map/components/route/hooks/DrawBestRoute.tsx

```diff
@@ -0,0 +1,66 @@
+import { useContext, useEffect, useReducer } from "react";
+import { AnySourceData, LngLatBounds, Map, Marker, Popup } from 'mapbox-gl';
+import { directionsApi } from "../../apis";
+import { DirectionsResponse } from "../../interfaces/direction";
+
+const getRouteBetweenPoints = async(start: [number, number], end: [number, number]) => {
+  const resp = await directionsApi.get<DirectionsResponse>(`/${start.join(',')};${end.join(',')}`);
+  const { distance, duration, geometry } = resp.data.routes[0];
+  const {coordinates: coords} = geometry;
+
+  let kms = distance / 1000;
+      kms = Math.round(kms * 100);
+      kms /= 100;
```

Comment:
Explain better what means those numbers, you can put them in a variable so it can be easy to read